From fadf7190eaadecbf2661fa70192d91edb84b45aa Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 28 Nov 2020 22:02:53 -0800
Subject: [PATCH] python3: fix compilation

Because of e52d0487e88c3c8c57e1310d1a02b18eae0d142e , these flags no
longer get passed. Add them back to fix compilation.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 lang/python/python3/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lang/python/python3/Makefile b/lang/python/python3/Makefile
index 294a636..c506e03 100644
--- a/lang/python/python3/Makefile
+++ b/lang/python/python3/Makefile
@@ -137,7 +137,8 @@ MAKE_FLAGS+=\
 EXTRA_CFLAGS+= \
 	-DNDEBUG -fno-inline
 EXTRA_LDFLAGS+= \
-	-L$(PKG_BUILD_DIR)
+	-L$(PKG_BUILD_DIR) \
+	-L$(STAGING_DIR)/usr/lib -L$(STAGING_DIR)/lib
 
 ENABLE_IPV6:=
 ifeq ($(CONFIG_IPV6),y)

